model/src/main/java/org/cbioportal/model/Gene.java

```diff
@@ -49,4 +51,26 @@ public Integer getLength() {
     public void setLength(Integer length) {
         this.length = length;
     }
+    
+    public String getChromosome() {
```

Comment:
@sheridancbio Thank you for bring this up. I didn't know there are genes on both X and Y. But I think in current get chromosome function, it will  recognize it as X since "Xp22.33; Yp11.3" only meet the condition that cytoband.toUpperCase().startsWith("X")